values.yaml

```diff
@@ -455,6 +455,9 @@ registry:
     #  requests:
     #    memory: 256Mi
     #    cpu: 100m
+    initContainerImage:
```

Comment:
You don't need a different init container image. You should simply reuse the `harbor-registry` or `harbor-registryctl` container image. This way, you also have the harbor username and group name, which means you can reference them directly instead of using the hardcoded UID and GID values.